ytmusicapi/parsers/browsing.py

```diff
@@ -54,13 +54,23 @@ def parse_content_list(results, parse_func, key=MTRIR):
 def parse_album(result):
     return {
         "title": nav(result, TITLE_TEXT),
-        "year": nav(result, SUBTITLE2, True),
```

Comment:
why remove this?

Reply:
YT may have moved something, on all the albums I've tried that nav route gives you the album's artist. I can't find year anywhere.

Reply:
This function has multiple usages (via `parse_mixed_content` (get_home, get_song_related), `parse_artist_contents` (get_artist, get_user)), please double-check that this is in fact returning `None` for all of them

Reply:
Looks like get_home's return is the same as get_album()["other_versions"], so it's also incorrectly returning artist names as the year. 

get_song_related and get_artist do correctly return year with the old version of parse_album, but could still benefit from type and audioplaylistid returns of the PR version.

get_user appears to be incapable of actually returning albums despite having access to the function, so I'm not sure on that one unless you have an ID I could test against. 

I'll rework the PR to flip between artists and year return based on the context.